tox.cover.sh

```diff
@@ -31,6 +31,8 @@ cover () {
     min=98
   elif [ "$1" = "certbot_dns_google" ]; then
     min=99
+  elif [ "$1" = "certbot_dns_nsone" ]; then
+    min=98
```

Comment:
nit: We can bump this to 99 to help ensure future changes have good test coverage.